tests/python/multidevice/test_deepseek_v3.py

```diff
@@ -120,19 +192,25 @@ def test_transformer_layer(setup_default_process_group):
         parallel_plan["mlp.shared_experts.up_proj"] = ColwiseParallel()
         parallel_plan["mlp.shared_experts.down_proj"] = RowwiseParallel()
 
-        transformer_layer = parallelize_module(
-            transformer_layer,
-            mesh,
-            parallel_plan,
-        )
-
-        # Sanity-check parameters are indeed distributed
-        distributed_params: list[str] = [
-            name
-            for name, parameter in transformer_layer.named_parameters()
-            if isinstance(parameter.data, DTensor)
-        ]
-        assert len(distributed_params) == 3 + (config.n_routed_experts + 1) * 3
+        match executor:
+            case Executor.TORCH_TP:
+                transformer_layer = parallelize_module(
+                    transformer_layer,
+                    mesh,
+                    parallel_plan,
+                )
+
+                # Sanity-check parameters are indeed distributed
+                distributed_params: list[str] = [
+                    name
+                    for name, parameter in transformer_layer.named_parameters()
+                    if isinstance(parameter.data, DTensor)
+                ]
+                assert len(distributed_params) == 3 + (config.n_routed_experts + 1) * 3
+            case Executor.NVFUSER:
+                parallelize_module_with_nvfuser(
+                    transformer_layer, mesh, parallel_plan, ""
+                )
```

Comment:
```Suggestion
parallelize_module_with_nvfuser(
                    transformer_layer, mesh, parallel_plan, fqn=""
                )
```

So I don't have to switch between functions to understand what the empty string indicates.